test/unit/ts.js

```diff
@@ -71,18 +71,19 @@ tap.test('TS Unit Tests', async t => {
     );
   }
 
+  const {generateOutputFileList, generateBundleObj} = await import(
```

Comment:
Why `await import` instead of an import statement at the top?

Reply:
Originally I did it to preserve some sort of visual spatial locality, updated now, however.